src/test/java/base/BaseTest.java

```diff
@@ -1,71 +1,71 @@
+import static utils.PropertiesHelper.getInstance;
+
 public class BaseTest {
-  public static WebDriver driver;
-  public static PropertiesHelper propertiesHelper;
-  protected static HomePage homePage;
-  protected static PageFactoryManager pageFactoryManager;
-  protected static long timeout;
-  protected static String baseUrl;
-  private static String hubUrl;
-  protected final int FIRST = 1;
+    public static WebDriver driver;
+    public static PropertiesHelper propertiesHelper;
+    protected static HomePage homePage;
+    protected static PageFactoryManager pageFactoryManager;
+    protected static String browser;
+    protected static boolean localRun;
+    protected static long timeout;
+    protected static String baseUrl;
+    private static String hubUrl;
+    protected final int FIRST = 1;
 
-  private static void configureLog4j() {
-    PropertyConfigurator.configure(propertiesHelper.getProperty("log4jPropertiesPath"));
-  }
+    private static void configureLog4j() {
+        PropertyConfigurator.configure(propertiesHelper.getProperty("log4jPropertiesPath"));
+    }
 
-  @Parameters({"browser", "localRun"})
-  @BeforeMethod
-  public void setUp(String browser, @Optional("true") boolean localRun) {
-    setUpDriver(browser, localRun);
-    driver.manage().window().maximize();
-    pageFactoryManager = new PageFactoryManager(driver);
-    homePage = pageFactoryManager.getPage(HomePage.class);
-    propertiesHelper = getInstance();
-    configureLog4j();
-    configureConstant();
-  }
+    @BeforeEach
+    public void setUp() {
+        propertiesHelper = getInstance();
+        configureConstant();
+        setUpDriver(browser, localRun);
+        driver.manage().window().maximize();
+        pageFactoryManager = new PageFactoryManager(driver);
+        homePage = pageFactoryManager.getPage(HomePage.class);
 
-  public void setUpDriver(String browser, boolean localRun) {
-    String envBrowser = System.getenv("BROWSER");
-    browser = ofNullable(envBrowser).orElse(browser);
-    WebDriverCreator creator;
-    if (localRun) {
-      creator = new LocalWebDriverCreator(browser);
-    } else {
-      creator = new RemoteWebDriverCreator(browser, hubUrl);
+        configureLog4j();
     }
-    driver = creator.createWebDriver();
-  }
 
-  public void configureConstant() {
-    hubUrl = propertiesHelper.getProperty("hubUrl");
-    baseUrl = propertiesHelper.getProperty("baseUrl");
+    public void setUpDriver(String browser, boolean localRun) {
+        String envBrowser = System.getenv("BROWSER");
+        browser = ofNullable(envBrowser).orElse(browser);
+        WebDriverCreator creator;
+        if (localRun) {
+            creator = new LocalWebDriverCreator(browser);
+        } else {
+            creator = new RemoteWebDriverCreator(browser, hubUrl);
+        }
+        driver = creator.createWebDriver();
+    }
 
-    timeout = Long.parseLong(propertiesHelper.getProperty("timeout"));
-  }
+    public void configureConstant() {
```

Comment:
There is a better approach, how to deal with properties. For example, you can create an `EnvirementContext` and put there all logic related to properties.